script/scrape_ntd.py

```diff
@@ -25,7 +27,9 @@
 )
 from pydantic import HttpUrl, parse_obj_as
 
-CALITP_BUCKET__NTD_DATA_PRODUCTS = os.environ["CALITP_BUCKET__NTD_DATA_PRODUCTS"]
+# CALITP_BUCKET__NTD_DATA_PRODUCTS = os.environ["CALITP_BUCKET__NTD_DATA_PRODUCTS"]
+# bucket: gs://calitp-ntd-data-products
+CALITP_BUCKET__NTD_DATA_PRODUCTS = "gs://test-calitp-ntd-data-products"
```

Comment:
Make sure to restore these environment variable references before merging, rather than using hardcoded values.

The actual hardcoded value can be defined in /airflow/docker-compose.yaml, where you'll find labeled environment variable definitions for many other variables used by scripts and jobs.

Reply:
I doubt this job will get actually put into airflow since the source data and paths change somewhat regularly.